packages/j/juliamono.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "juliamono-font",
+  "description": "A monospaced font with reasonable Unicode support",
+  "filename": "juliamono.css",
```

Comment:
```suggestion
```
It looks like you're only distributing the font files, so you can probably just omit the default filename?

Reply:
Thanks - I wasn't quite sure about this... :)